utils/providers.py

```diff
@@ -49,7 +49,7 @@ def list_providers(allowed_types=None):
     return providers
 
 
-def get_mgmt(provider_key, providers=None, credentials=None):
+def get_mgmt(provider_key, providers=None, credentials=None, version=None):
```

Comment:
Version shouldn't ever be passed to ```get_mgmt```. This function is used for reading the yamls and "getting the" the version from there. Where you _could_ support it is in the mgmt_class, if it isn't already,